sql/core/src/test/scala/org/apache/spark/sql/columnar/ColumnTypeSuite.scala

```diff
@@ -197,8 +197,8 @@ class ColumnTypeSuite extends SparkFunSuite with Logging {
   }
 
   def testNativeColumnType[T <: AtomicType](
-      columnType: NativeColumnType[T],
-      putter: (ByteBuffer, T#InternalType) => Unit,
+      columnType: NativeColumnType[T])
+      (putter: (ByteBuffer, T#InternalType) => Unit,
```

Comment:
we can use curry here to infer the type `T#InternalType`, thus no need to add type parameter explicitly like `testNativeColumnType[IntegerType.type](INT, _.putInt(_), _.getInt)`.